### ISSUE TITLE
'import datalad' fails on Windows

### ISSUE THREAD

**Issue by mih** [datalad/datalad#1137]
I installed git. git-annex, python 2.7, and then datalad via pip. All worked flawless. But then this:

````
Python 2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:19:22) [MSC v.1500 32 bit (
Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import datalad
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python27\lib\site-packages\datalad\__init__.py", line 21, in <module>

    ssh_manager = SSHManager()
  File "C:\Python27\lib\site-packages\datalad\support\sshconnector.py", line 189, in __init__
    not_supported_on_windows("TODO: Make this an abstraction to "
  File "C:\Python27\lib\site-packages\datalad\utils.py", line 103, in not_supported_on_windows
    + (": %s" % msg if msg else ""))
NotImplementedError: This functionality is not yet implemented for Windows OS: TODO: Make this an abstraction to interface platform dependent SSH
````

**Comment by yarikoptic** [comment]
I really doubt that any notable functionality of datalad could be achieved on windows ATM without much of investment... even direct mode support is not yet flashed out AFAIK

**Comment by mih** [comment]
Gotta start somewhere...

**Comment by mih** [comment]
FTR: This is also a problem when using git-annex directly on windows with datalad datasets, as it will try to enable the datalad special remote and fail with a page of errors due to due import failure.

**Comment by yarikoptic** [comment]
oh... Would you mind pasting errors due to due?

**Comment by chrisfilo** [comment]
Just run into this one as well.

**Comment by mih** [comment]
This is no longer an issue in https://github.com/mih/datalad/tree/tryme.

Import works, `--help` works. Running anything currently depends on a fix for #1565
